modules/core/04-channel/v2/types/keys.go

```diff
@@ -13,5 +15,11 @@ const (
 // AsyncPacketKey returns the key under which the packet is stored
 // if the receiving application returns an async acknowledgement.
 func AsyncPacketKey(clientID string, sequence uint64) []byte {
-	return []byte(fmt.Sprintf("%s/%s/%d", KeyAsyncPacket, clientID, sequence))
```

Comment:
Note, I changed the key format to make it the same style as the other ICS24 keys